core-models/src/main/scala/com/mesosphere/usi/core/models/Ids.scala

```diff
@@ -0,0 +1,5 @@
+package com.mesosphere.usi.core.models
```

Comment:
Hm, now this is interesting. Should we put the ids in the ids package? I'd say so. This would yield a stuttering api, though: `com.mesosphere.usi.core.models.ids.PodId`. What do you think?

Reply:
I don't like too granular packages, I think we can always refactor later if we notice that we have way too many things there.